resolwe_bio/processes/variant_calling/gatk4_hc.py

```diff
@@ -158,6 +164,9 @@ def run(self, inputs, outputs):
         if inputs.intervals_bed:
             args.extend(["-L", inputs.intervals_bed.output.bed.path])
 
+            if inputs.advanced.interval_padding:
```

Comment:
Since the default is set to `100`, this option will always be enabled. I don't think this was as intended, correct?

Reply:
I made a change so that `interval_padding` is not required.